addons/misra.py

```diff
@@ -1833,7 +1833,10 @@ def reportErrorIfVariableIsNotConst(variable, stringLiteral):
                         continue
 
                     parametersUsed = getArguments(token)
-                    for i in range(len(parametersUsed)):
+
+                    checkArgCount = len(functionDeclaration.argument) - 1 if len(parametersUsed) > len(functionDeclaration.argument) else len(parametersUsed)
```

Comment:
can you please use min/max instead of `if else` here..

Reply:
I wonder if this would work:

    checkArgCount = min(len(functionDeclaration.argument) - 1, parametersUsed)